tests/API/XCCDF/default_cpe/test_default_cpe.sh

```diff
@@ -6,45 +6,31 @@
 . $builddir/tests/test_common.sh
 
 CPE_OVAL="cpe-oval.xml"
-FEDORA_XCCDF="fedoraX-xccdf.xml"
-FEDORA_CPE="cpe:/o:fedoraproject:fedora:"
-RHEL_XCCDF="rhelX-xccdf.xml"
-RHEL_CPE="cpe:/o:redhat:enterprise_linux:"
+CPE_DICT="cpe-dict.xml"
+XCCDF="xccdf.xml"
 SYSTEM_CPE_FILE="/etc/system-release-cpe"
+SYSTEM_CPE=$(cat $SYSTEM_CPE_FILE)
```

Comment:
Do I understand it well that the "cpe-oval.xml" in this directory always passes? If yes, why to bother with system CPE? Instead, you can hardcode a random dummy CPE into this test. wdyt?

Reply:
Great idea, I will do it like that, thank you.